docs/src/app/components/pages/components/SelectField/ExampleCustomLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import SelectField from 'material-ui/lib/SelectField';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+
+export default class SelectFieldCustomLabelExample extends React.Component {
```

Comment:
Then we should also invert the ordre in the example :grin:. I think that it's better to be more end more specific in the name.